docs/contributing/docker-for-testing.rst

```diff
@@ -55,38 +55,31 @@ Testing on an unmerged branch
 
   .. code::
 
-    sudo docker build --tag=tripaldocker:testing-9999 --build-arg drupalversion="10.2.x-dev" --build-arg postgresqlversion="15" --file tripaldocker/Dockerfile-php8.3 ./
+    sudo docker build --tag=tripaldocker:testing-9999 --build-arg drupalversion="10.2.x-dev" --build-arg postgresqlversion="15" --build-arg phpversion=8.3 ./
```

Comment:
We may be able to update this to `drupalversion="10.2.x-dev"` and `postgresqlversion="16"` now

Reply:
Ahh! Good catch -thank you ❤️